### PR TITLE
Nav install

### PR DESCRIPTION
**Pull request by krab7191**

### CODE REVIEW

File: client/src/components/DropDownNav/DropDownNav.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <div>
+        <Button
+          aria-owns={anchorEl ? 'simple-menu' : undefined}
+          aria-haspopup="true"
+          onClick={this.handleClick}
+        
+        >
+          
+            <MenuIcon style={{height: '40px', width: '40px'}} />
+
```
Comment (krab7191) [inline client/src/components/DropDownNav/DropDownNav.js:35]: Remove extra empty space

File: client/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -6,15 +6,15 @@ import { withStyles } from '@material-ui/core/styles';
...
+    backgroundColor: 'black',
     opacity: '0.4',
     fontSize: '15px',
     color: 'white',
     borderTop: '1px solid #E7E7E7',
     textAlign: 'left',
     left: '0',
     bottom: '0',
-    height: '70px',
+    height: '20px',
```
Comment (krab7191) [inline client/src/components/Footer/Footer.js:17]: Lets get in the habit of using REM for almost everything for responsiveness

File: client/src/components/Nav/Nav.js [hunk 3]
```diff
@@ -1,22 +1,40 @@
...
-// import { Link } from 'react-router-dom'
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom'
+import DropDownNav from '../DropDownNav'
 import './Nav.css'
+// import { orange } from '@material-ui/core/colors';
 class Nav extends React.Component {
   state = {
   
   };
```
Comment (krab7191) [inline client/src/components/Nav/Nav.js:10]: Remove unused state

File: client/src/components/Nav/Nav.js [hunk 4]
```diff
@@ -1,22 +1,40 @@
...
+        </div>
+        <div className="dropdown">
+        <DropDownNav />
+         </div>
+          {/* <a href="#">Link 2</a>
+          <a href="#">Link 3</a>
+
+          {/* dropdown menu */}
+          
+
```
Comment (krab7191) [inline client/src/components/Nav/Nav.js:37]: Remove commens, empty space, format file. Maybe should prioritize Lint config @ocskier ?

File: client/src/pages/MapPage/MapPage.js [hunk 5]
```diff
@@ -17,6 +17,9 @@ class MapPage extends Component  {
  
   return (
     <div>
+    <h1>
+      <br></br><br></br>
```
Comment (krab7191) [inline client/src/pages/MapPage/MapPage.js:21]: Tons of line breaks is discouraged; harder to maintain. Favor margin.

File: client/src/pages/home/home.js [hunk 6]
```diff
@@ -38,7 +38,7 @@ function Home(props) {
         <Grid item xs={12}></Grid>
         <Grid item sm={12}>
           <Paper className={classes.paper}>
-            <h1>Find Local Farms</h1>
+            <br></br><br></br>
```
Comment (krab7191) [inline client/src/pages/home/home.js:41]: Same here, favor margin

**Comment by krab7191** [comment]
@here

**Comment by tlburchh** [comment]
@krab7191 I don't want to merge this until I know that I am merging it to the right place

**Pull request merged**